test/services/XmlValidationServiceSpec.scala

```diff
@@ -34,27 +36,12 @@ class XmlValidationServiceSpec extends SpecBase {
       }
 
       "with an enroute event" in {
-        val xml = buildXml(withEnrouteEvent = true)
 
-        xmlValidationService.validate(xml, ArrivalNotificationXSD) mustBe a[Right[_, _]]
-      }
-
-      "with an enroute event and incident" in {
-        val xml = buildXml(withEnrouteEvent = true, withIncident = true)
-
-        xmlValidationService.validate(xml, ArrivalNotificationXSD) mustBe a[Right[_, _]]
-      }
-
-      "with an enroute event and container transhipment" in {
-        val xml = buildXml(withEnrouteEvent = true, withContainerTranshipment = true)
-
-        xmlValidationService.validate(xml, ArrivalNotificationXSD) mustBe a[Right[_, _]]
-      }
-
-      "with an enroute event and vehicular transhipment" in {
-        val xml = buildXml(withEnrouteEvent = true, withVehicularTranshipment = true)
-
-        xmlValidationService.validate(xml, ArrivalNotificationXSD) mustBe a[Right[_, _]]
+        forAll(arbitrary[Boolean], arbitrary[Boolean], arbitrary[Boolean], arbitrary[Boolean]) {
```

Comment:
The`withIncident `  is not being used, which means we're not testing this branch of logic, I think we either break this out into two tests: 

1. "with an enroute event without incident"
2. "with an enroute event with incident"

or make the test with arbitrary `withIncident` value work

Reply:
@amaalali Good catch, we will look into it